README.md

```diff
@@ -241,6 +241,23 @@ associative array in the first argument:
 </div>
 ```
 
+You can chain calls if you prefer as there's also a `stimulus_controller` filter:
```

Comment:
Let's go further :). I think we should remove the documentation for the "old" entirely. It's quite ugly and hard to understand. Eventually, we should even deprecate it.